perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py

```diff
@@ -198,18 +278,18 @@ def _execute_pkb_cmd(pkb_cmd, stdout_filename, stderr_filename):
   # Will probably have to implement with threading.
   p.wait()
   elapsed_time = time.time() - start_time
+  if elapsed_time == 1:
```

Comment:
that's very fragile. We should check the status of the run by looking at the expected output and/or artifacts, rather than the duration. The duration could change for any reason. Also, doing equality comparison (==1) makes it even more fragile and usually this kind of hack should at least be an inequality comparison (e.g., < 3 or such) but I am not in favor of any one of these.